root/report/components/ReleaseList.js

```diff
@@ -16,42 +16,73 @@ import type {ReportReleaseT} from '../types';
 import ArtistCreditLink
   from '../../static/scripts/common/components/ArtistCreditLink';
 
+type Props = {|
+  +items: $ReadOnlyArray<ReportReleaseT>,
+  +pager: PagerT,
+  +showLanguageAndScript?: boolean,
+|};
+
 const ReleaseList = ({
   items,
   pager,
-}: {items: $ReadOnlyArray<ReportReleaseT>, pager: PagerT}) => (
-  <PaginatedResults pager={pager}>
-    <table className="tbl">
-      <thead>
-        <tr>
-          <th>{l('Release')}</th>
-          <th>{l('Artist')}</th>
-        </tr>
-      </thead>
-      <tbody>
-        {items.map((item, index) => (
-          <tr className={loopParity(index)} key={item.release_id}>
-            {item.release ? (
-              <>
-                <td>
-                  <EntityLink entity={item.release} />
-                </td>
-                <td>
-                  <ArtistCreditLink
-                    artistCredit={item.release.artistCredit}
-                  />
-                </td>
-              </>
-            ) : (
-              <td colSpan="2">
-                {l('This release no longer exists.')}
-              </td>
-           )}
+  showLanguageAndScript,
+}: Props) => {
+  const colspan = showLanguageAndScript ? 3 : 2;
```

Comment:
Following other comment:
```suggestion
  const colSpan = showLanguageAndScript ? 3 : 2;
```